README.md

```diff
@@ -39,18 +39,18 @@ MLStyle.jl
 
 ## What's MLStyle.jl?
 
-MLStyle.jl is a Julia package that provides multiple productivity tools from ML([Meta Language](https://en.wikipedia.org/wiki/ML_(programming_language))) like [pattern matching](https://en.wikipedia.org/wiki/Pattern_matching) that're statically generated and extensible, ADTs/GADTs([Algebraic Data Type](https://en.wikipedia.org/wiki/Algebraic_data_type), [Generalized Algebraic Data Type](https://en.wikipedia.org/wiki/Generalized_algebraic_data_type)) and [Active Patterns](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/active-patterns).
+MLStyle.jl is a Julia package that provides multiple productivity tools from ML([Meta Language](https://en.wikipedia.org/wiki/ML_(programming_language))) like [pattern matching](https://en.wikipedia.org/wiki/Pattern_matching) that is statically generated and extensible, ADTs/GADTs([Algebraic Data Type](https://en.wikipedia.org/wiki/Algebraic_data_type), [Generalized Algebraic Data Type](https://en.wikipedia.org/wiki/Generalized_algebraic_data_type)) and [Active Patterns](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/active-patterns).
 
-If you still have problems with the scoping of MLStyle.jl, treat it as **FP.jl**.
+Think of MLStyle.jl as a packaging bringing advanced functional programming idioms to julia.
 
 
 ## Motivation
 
-The people who're used to so-called functional programming could become retarded when there're no pattern matching and ADTs, and of course I'm one of them.
+Those used to functional programming may feel limited when they don't have pattern matching and ADTs, and of course I'm one of them.
```

Comment:
Okay, I was trying to play a Chinese meme when writing this down... However, it seems not that work in English...

Reply:
Yeah I could tell it was a playful joke, but its also the sort of joke many english speakers would find offensive so I figured it may be safer to cut it.